src/components/tag-input/tag-input.component.less

```diff
@@ -11,6 +11,7 @@ ux-tag-input {
         list-style-type: none;
         margin: 0;
         padding: 1px 3px;
+        padding-bottom: 0;
```

Comment:
Think this needs to be reverted as well, that should sort out the height changes.